viewer/src/main/webapp/viewer-html/components/components.json

```diff
@@ -672,6 +672,18 @@
         showHelp: true
     },
     {
+        className: "viewer.components.Snapping",
+        name: "Snapping",
+        shortName: "Sn",
+        group: "Component",
+        sources: ["OpenLayersSnappingController.js", "Snapping.js"],
```

Comment:
nog toevoegen dat het (voor nu) alleen openlayers is:
        notInCombinationWith: ["viewer.mapcomponents.FlamingoMap"]